sdcm/ec2_client.py

```diff
@@ -83,6 +85,7 @@ def _request_spot_instance(self, instance_type, image_id, region_name, network_i
                                            },
                       ValidUntil=datetime.datetime.now() + datetime.timedelta(minutes=self._timeout/60 + 5)
                       )
+        self.add_placement_group_name_param(params['LaunchSpecification'])
```

Comment:
still I don't like idea of leaking SCT specifics to `ec2_client` - I'd recommend providing `placement_group_name` as an argument to `_request_spot_instance` and do the proper logic when it is set.

Reply:
fixed. now  placement_group_name is provided as an argument